docs/providers/aws/guide/serverless.yml.md

```diff
@@ -16,117 +16,259 @@ layout: Doc
 
 Here is a list of all available properties in `serverless.yml` when the provider is set to `aws`.
 
+## Root properties
+
 ```yml
 # serverless.yml
 
-service: myService
+# Service name
+service: myservice
 
+# Framework version constraint: '2', '>=2.33', '^2.33'
 frameworkVersion: '2'
-configValidationMode: warn # Modes for config validation. `error` throws an exception, `warn` logs error to console, `off` disables validation at all. The default is warn.
-enableLocalInstallationFallback: false # If set to 'true', guarantees that it's a locally (for service, in its node_modules) installed framework which processes the command
-useDotenv: false # If set to 'true', environment variables will be automatically loaded from .env files
-variablesResolutionMode: null # To crash on variable resolution errors (as coming from new resolver), set this value to "20210326"
-unresolvedVariablesNotificationMode: warn # If set to 'error', references to variables that cannot be resolved will result in an error being thrown (applies to legacy resolver)
-deprecationNotificationMode: warn:summary # 'warn' reports deprecations on the go, 'error' will result with an exception being thrown on first approached deprecation
-
-disabledDeprecations: # Disable deprecation logs by their codes. Default is empty.
+
+# Configuration validation: 'error' (fatal error), 'warn' (logged to the output) or 'off' (default: warn)
+# See https://www.serverless.com/framework/docs/configuration-validation
+configValidationMode: error
+# Run the local 'serverless' package if it exists in 'node_modules'
+enableLocalInstallationFallback: false
+# Load environment variables from .env files (default: false)
+# See https://www.serverless.com/framework/docs/environment-variables
+useDotenv: true
+# Turn variable resolution errors in strict fatal errors by setting this value to "20210326" (default: null)
+variablesResolutionMode: 20210326
+# If set to 'error', references to variables that cannot be resolved will result in an error being thrown (applies to legacy resolver)
+unresolvedVariablesNotificationMode: warn
+# 'warn' reports deprecations on the go, 'error' will result with an exception being thrown on first approached deprecation
+deprecationNotificationMode: warn:summary
+# Disable deprecations by their codes (default: empty)
+# See https://www.serverless.com/framework/docs/deprecations
+disabledDeprecations:
   - DEP_CODE_1 # Deprecation code to disable
   - '*' # Disable all deprecation messages
+```
+
+## Provider
+
+### General settings
+
+```yml
+# serverless.yml
 
 provider:
   name: aws
+  # Default stage (default: dev)
+  stage: dev
+  # Default region (default: us-east-1)
+  region: us-east-1
+  # The AWS profile to use to deploy
```

Comment:
I want to make this change:

```suggestion
  # The AWS profile to use to deploy (default: "default" profile)
```

Could anyone confirm this is correct?

Reply:
yes, by default we're using `default` AWS profile